common/scala/src/main/scala/whisk/connector/kafka/KafkaMessagingProvider.scala

```diff
@@ -0,0 +1,37 @@
+package whisk.connector.kafka
+
+import akka.actor.ActorSystem
+import scaldi.Injector
+import whisk.common.Logging
+import whisk.core.WhiskConfig
+import whisk.core.connector.MessageConsumer
+import whisk.core.connector.MessageProducer
+import whisk.core.connector.MessagingProvider
+import whisk.spi.SpiFactoryModule
+/**
+  * Created by tnorris on 6/20/17.
+  */
+class KafkaMessagingProvider(actorSystem:ActorSystem, config:WhiskConfig)(implicit logging:Logging) extends MessagingProvider {
```

Comment:
Not sure what you mean - you want to change the name from "KafkaMessagingProvider" to "Kafka"?

There is already a "MessagingProvider" trait defined (which is the one that extends Spi):
https://github.com/apache/incubator-openwhisk/pull/2414/files#diff-8ba49f030237d2d826b76d762a468013R9